server/src/test-data/fixtures.js

```diff
@@ -203,11 +203,31 @@ export const GROUPS = [
       'PAR',
     ],
   },
+  {
+    name: 'Metro Zone',
+    icon: 'group',
+    users: [
+      'parramatta5',
+    ],
+    tags: [
+      'NSW',
+    ],
+  },
+  {
+    name: 'OCES Working Group',
+    icon: 'mci-radio-handheld',
+    users: [
+      'parramatta3',
+    ],
+    tags: [
+      'NSW',
+    ],
+  },
 ];
 
 export const SCHEDULES = [
   {
-    name: 'Wagga Wagga OOA',
+    name: 'Wagga Wagga OOA availability',
```

Comment:
red flag any time I see a constant string appended/prepended to all entries.. I wonder if we should build it into the app so everything always says availability after it? Or do you see situations where we put something other than availability there? No changes required to this diff but we should discuss.

Reply:
yes, totally agree. the UI should be handing this but I need a quick win for the Wednesday demo :P